k9mail/src/test/java/com/fsck/k9/message/html/HtmlSanitizerTest.java

```diff
@@ -264,4 +264,17 @@ public void shouldKeepUris() {
                 "<a href=\"rtsp://example.com/media.mp4\">RTSP</a>" +
                 "</body></html>", toCompactString(result));
     }
+
+    @Test
+    public void shouldKeepDirAttribute() {
+        String html = "<html><body>" +
+                "<table><tbody><tr><td dir=\"rtl\"></td></tr></tbody></table>" +
+                "</body></html>";
+
+        Document result = htmlSanitizer.sanitize(html);
+
+        assertEquals("<html><head></head><body>" +
```

Comment:
Wouldn't `assertEquals(html, toCompactString(result))` better communicate our expectation? (I realize the input needs to be changed then)